include/IrrCompileConfig.h

```diff
@@ -1,6 +1,6 @@
-// Copyright (C) 2002-2012 Nikolaus Gebhardt
-// This file is part of the "Irrlicht Engine".
-// For conditions of distribution and use, see copyright notice in irrlicht.h
+// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
```

Comment:
this old file you need to mention it came from irrlicht originally

Reply:
but should probably be split into separate compile config includes for every namespace (core,asset,video, etc.)

Reply:
TODO

Reply:
@Przemog1 this is the final thing holding up the merge